cms_status_message/models.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# © 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models, api
+from openerp.http import request
+from openerp import _
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    @property
```

Comment:
added tests for translations + test: it works ;)
Only one doubt: if you customize translations in your own module this test is going to fail?
In my experience this is true for instance with tour tests: if you remove the "add content" menu in your theme, the tour that test page add is going to fail and there's no way to deactivate it than patch the phantom test.